article/ArticleStatusController.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+var bodyParser = require('body-parser');
+const ArticleStatus = require('../data/models/ArticleStatus');
+
+const router = express.Router();
+router.use(bodyParser.urlencoded({ extended: true }));
+router.use(bodyParser.json());
+
+router.get('/', async function(req, res) {
+  try {
+    const articles = await ArticleStatus.scan().exec();
+    res.send(articles);
+  } catch (err) {
+    console.error(err);
+    res.sendStatus(500);
+  }
+});
+
+router.post('/', async function(req, res) {
```

Comment:
Should the alexa skill call this too?  Or should we use this.attributes['offsetMilliseconds']?  Does it matter?

Reply:
Yes, the skill should use this too so that you can move back & forth between Alexa & mobile.